docs/modules/ROOT/pages/references/functional-requirements/billing.adoc

```diff
@@ -21,64 +21,111 @@ A list of features the system should have:
 
 .Example invoice for APPUiO Cloud
 ====
-_Invoice period: 1.1. - 31.1. (44'640 minutes, timezone UTC)._
-
-[cols="4,1,1,1,1,1", options="header", stripes="hover"]
+[cols="4,1,1,1", options="header", stripes="hover"]
 |===
 |Description
 |Quantity
-|Unit of Measure
-|Unit Price
-|Discount
-|Total (Rounded)
+|Price
+|Total
 
-6+|*Zone: cloudscale.ch LPG 2 - Namespace: my-app-prod*
+4+|*Zone: cloudscale.ch - LPG 2 - Namespace: my-app-prod*
 
 a|Compute
 
-Min: 128 MiB +
-Max: 1034 MiB +
-Avg: 585.66 MiB
+Qty: 16602449.13 MiB-minutes +
+Unit Price: CHF 0.0000012432 / MiB / Minute
+
+|1
+|CHF 20.64
+>|CHF 20.64
+
+a|Persistent Storage (PVC)
+
+Storage Type: RWX +
+Qty: 130500.00 GiB-minutes +
+Unit Price: 0.0000710000 / GiB / Minute
+
+|1
+|CHF 9.27
+>|CHF 9.27
+
+a|Persistent Storage (PVC)
+
+Storage Type: RWO (SSD) +
```

Comment:
> RWO (SSD)

Could that be just name of the StorageClass? In this case, the values would be `ssd`, `bulk`, `cephfs-fspool-cluster` on Cloudscale and `rbd-storagepool-cluster` and `cephfs-fspool-cluster` on Exoscale.

If other values would be desired, we would have to add a mapping into the adapter.

Reply:
Why not just add a db field 'description' or 'invoice_description' in the products table?

Reply:
Still, something we would have to manage when ever we add a new storage class on a cluster. Also, I still keep in mind that another ERP might be added. I want to prevent cluttering the core with ERP specific features. So very reluctant to add that field.